README.md

```diff
@@ -32,6 +32,14 @@ for notes on how to deploy the project on a live system.
   npm install -g gitdocs@next
   ```
 
+- Typescript
+
+  TypeScript is a dependency used in EddieBotUI , so you need to install it first.
+
+  ```bash
+  npm install typescript
+  ```
+
```

Comment:
Thank you for adding this 👍

Reply:
I think we might not need this if we add `typescript` to the `package.json` in the `devDependencies` using the command 

```
npm install --save-dev typescript`
```

This will add it to the `package.json` so that in the future when someone runs `npm install` it will be included

Reply:
yes, you're right Eddie. That actually ends up being easier for someone getting started 👍